### DISCUSSION TITLE
Error when embedding Observable code

### DISCUSSION THREAD

**Discussion by jhjanicki** [quarto-dev/quarto-cli#4262] [Q&A]
Hello!
I created a Quarto document in R and when I tried to include Observable code by using the following blocks  ```{ojs} ... ``` I got this error: /bin/sh: ojs: command not found
I am using a Mac.  Is there something I would need to install in order to use Observable code in Quarto?

**Comment by cscheid** [comment]
Can you be more specific about what you mean by "I created a Quarto document in R" and "tried to include Observable"? We develop on Macs and we don't see this issue.

I'm not sure how you'd have gotten that error message. Could it be you tried to hit "Run" on an OJS cell in RStudio? That won't work; OJS cells need a full webpage. You need to hit "Render".
